docs/pages/errors/index.vue

```diff
@@ -0,0 +1,218 @@
+      </p>
+      <DocsDoNot>
+        <template v-slot:do>
+          <p class="do-dont">
+            “Sorry, something went wrong. <u>Refresh page</u>"
+          </p>
+        </template>
+        <template v-slot:not>
+          <p class="do-dont">
+            “500 internal server error. It looks like the server failed to fulfill an apparently valid GET or POST request. We are aware of this issue and you can try solving it by reloading the page. We apologize for the inconvenience and hope this doesn’t occur again.”
+          </p>
+        </template>
+      </DocsDoNot>
+      <h3>
+        Tone
+      </h3>
+      <p>
+        Let the user know what went wrong without blaming them. Do not use all caps, exclamation points, or other alarmist language.
+      </p>
+      <DocsDoNot>
+        <template v-slot:do>
+          <p class="do-dont">
+            “Please enter a valid email”
+          </p>
+        </template>
+        <template v-slot:not>
+          <p class="do-dont">
+            “WARNING!! You didn’t fill this out”
+          </p>
+        </template>
+      </DocsDoNot>
+      <p>
+        Even though errors can induce negative emotions, avoid being too whimsical and lighthearted. This can have the opposite effect and cause more frustration.
+      </p>
+      <DocsDoNot>
+        <template v-slot:do>
+          <p class="do-dont">
+            “Storage limit reached”
+          </p>
+        </template>
+        <template v-slot:not>
+          <p class="do-dont">
+            “Whoops! Looks like the tank is too full &#x1f60e;"
+          </p>
+        </template>
+      </DocsDoNot>
+    </DocsPageSection>
+
+    <DocsPageSection title="Components and patterns" anchor="#components">
+      <h3>
+        Color
+      </h3>
+      <DocsColorBlock name="tokens.error" definition />
+      <h3>
+        Icon
+      </h3>
+      <KIcon icon="error" />
+      <p>
+        Sizes: 14px, 24px
+      </p>
+      <h3>
+        Banner
+      </h3>
+      <p>
+        Banners are appended to the app bar and notify the user about global system errors and/or notifications that affect the whole app experience.
+      </p>      
+      <img src="./banner.png">
+      <h3>
+        Error summary
+      </h3>
+      <p>
+        Error summaries should appear at the top of the page and use an alert component. Use them in forms or for page-level errors that are not associated with a specific component or location on the page. 
+      </p>
+      <img src="./alert.png">
+      <h3>
+        Inline error messages
+      </h3>
+      <p>
+        Place inline error messages in close proximity to where the error occurred. This can be an icon only, or icon and text. When paired with text, use <code>KLabeledIcon</code>
```

Comment:
This won't work until we add a docs page for it, but when we do we'll be able to use `DocsInternalLink` with `code` attribute:


```html
<DocsInternalLink code href="/klabeledicon">KLabeledIcon</DocsInternalLink>
```